terraform/elasticsearch.tf.json.template.py

```diff
@@ -108,6 +107,6 @@
                     }
                 }
             }
-        }
+        } if domain else {}
```

Comment:
This seems like an unrelated change. Can you explain why you added this?

Reply:
Does this solve the [0] issue with the aws_eslaticsearch_domain name?

Reply:
Yes -- Hannes requested this in the original ticket.